main.py

```diff
@@ -295,37 +299,71 @@ async def command_list(message: Message):
     # Scrape all tracks in the target channel and list them
     channel_media_attachments = await scrape_channel_media(target_channel)
 
+    # title of !list embed
     embed_title = "❤️‍🔥 AIGHT. IT'S BUSTY TIME ❤️‍🔥"
-    embed_content = "**Track Listing**\n"
+    embed_description_prefix = "**Track Listing**\n"
+
+    # stack of embed descriptions to circumvent the 4096 character embed limit
+    embed_description_stack = []
+    embed_description_current = ""
 
     for index, (
         submit_message,
         attachment,
         local_filepath,
     ) in enumerate(channel_media_attachments):
         list_format = "**{0}.** {1}: [{2}]({3}) [`↲jump`]({4})\n"
-        embed_content += list_format.format(
+        list_entry = list_format.format(
```

Comment:
```suggestion
        song_list_entry = list_format.format(
```

I thought this was referring to the list of embeds for a moment.